profiles/civicrm_starterkit/patches/pantheon-settings-starterkit-55.patch

```diff
@@ -17,9 +17,10 @@ index 34aaafe59d..9ef54a2fec 100644
 +*/
 +if (!empty($_SERVER['PRESSFLOW_SETTINGS'])) {
 +  $env = json_decode($_SERVER['PRESSFLOW_SETTINGS'], TRUE);
-+  if (!empty($env['conf']['pantheon_binding'])) {
++  if (!empty($env['conf']['pantheon_environment'])) {
 +    $pantheon_db = $env['databases']['default']['default'];
 +    $pantheon_conf = $env['conf'];
++    $pantheon_root = $_ENV['HOME'] . ($_ENV['HOME'] !== '/' ? '/' : '')
```

Comment:
I'm not clear on the point of this line or if it will work properly.

1. Isn't it better to output whatever `$_ENV['HOME']` is providing? It'll either be `/` or `/srv/bindings/[uuid]` and I assume that variable will match the actual environment and actually just work?
2. It has terniary where if the condition fails it will provide an empty string. I don't see how that would work as a fallback.

Reply:
Oh, I just read that line again. Now it makes sense to me.